tests/fuzz/fuzz-bolt12-tlvspan.c

```diff
@@ -0,0 +1,23 @@
+#include "config.h"
+#include <common/utils.h>
+#include <wire/wire.h>
+#include <tests/fuzz/libfuzz.h>
+
+/* Include bolt12.c directly, to gain access to tlv_span(). */
+#include "../../common/bolt12.c"
+
+void init(int *argc, char ***argv)
+{}
+
+void run(const u8 *data, size_t size)
+{
+	if (size < 2 * sizeof(u64))
+		return;
```

Comment:
Since `fromwire_u64` automatically handles the case where `size` is too small, I think we can remove this check.

```suggestion
```